docs/project-requirements.md

```diff
@@ -236,32 +236,38 @@ W każdym story pomijamy aktora, jest napisane w nagłówku kto nim jest. Dodatk
 # Kryteria akceptacji projektu dla I semestru prac
 
 ## Wymagane
-- Wybór gracza do pojedynku (US. 1.2.2)
-- Pojedynek między dwoma graczami (US. 1.1.8, 1.1.9)
-- Talia Kart (US. 1.1.13, 1.4.2 - 1.4.7, 1.1.15)
-- Profil Użytkownika (US. 1.4.1, 1.4.2, 1.1.5, 1.1.6, 1.1.14)
+- Pojedynek między dwoma graczami - symulator

```

Comment:
To bym przeniósł do "Oczekiwane"

Reply:
Zgadzam się, ta symulacja pojedynku do oczekiwanych